config/dbconfig.txt

```diff
@@ -3,31 +3,31 @@
 ## administration, and the in game library.
 
 ## Should SQL be enabled? Uncomment to enable
-#SQL_ENABLED
+SQL_ENABLED
```

Comment:
crinky cronge

Reply:
the hilarious thing is this makes the server just straight up hang, probably CI as well